packages/api-explorer/src/Params.jsx

```diff
@@ -150,17 +151,17 @@ Params.defaultProps = {
 };
 
 function createParams(oas) {
-  const ArrayField = createArrayField(oas);
-  const BaseInput = createBaseInput(oas);
-  const DateTimeWidget = createDateTimeWidget(oas);
-  const FileWidget = createFileWidget(oas);
-  const SchemaField = createSchemaField();
-  const SelectWidget = createSelectWidget(oas);
-  const TextareaWidget = createTextareaWidget(oas);
-  const URLWidget = createURLWidget(oas);
-
   // eslint-disable-next-line react/display-name
   return props => {
+    const ArrayField = createArrayField(oas, props.explorerEnabled);
+    const BaseInput = createBaseInput(oas, props.explorerEnabled);
+    const DateTimeWidget = createDateTimeWidget(oas, props.explorerEnabled);
+    const FileWidget = createFileWidget(oas, props.explorerEnabled);
+    const SchemaField = createSchemaField();
+    const SelectWidget = createSelectWidget(oas, props.explorerEnabled);
+    const TextareaWidget = createTextareaWidget(oas, props.explorerEnabled);
+    const URLWidget = createURLWidget(oas, props.explorerEnabled);
```

Comment:
@erunion not sure why it's saying I'm missing validation for these props :(

![image](https://user-images.githubusercontent.com/41130056/93399552-aea00f80-f832-11ea-9667-9635324abe7a.png)

but I have them on the [Params Component](https://github.com/readmeio/api-explorer/pull/940/files#diff-3c697e1c364b5422881949b233f9aa3cR135) and on each of the different widget components:
* [Response](https://github.com/readmeio/api-explorer/pull/940/files#diff-3603347a29fff625128f5ffc722c3759R75)
* [Response Example](https://github.com/readmeio/api-explorer/pull/940/files#diff-99031c112c99dd4e1b4a1bda546ffc7bR275)
* [Array Field](https://github.com/readmeio/api-explorer/pull/940/files#diff-a96510858c4571c0e832e75c83da461bR49)
and so on

Reply:
It might be because it's coming through as just an object on line 155 and ESLint doesn't see it as being part of the `Params` component props. To fix it you can probably save `props.explorerEnabled` to a local `explorerEnabled` constant in `createParams`, silence the error with `// eslint-disable-next-line react/prop-types` and then pass `explorerEnabled` into these creation calls.

Reply:
did some refactoring. so close to passing tests 😞